modules/oe_starter_content_person/config/install/core.entity_form_display.node.oe_sc_person.default.yml

```diff
@@ -47,6 +49,24 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  oe_sc_person_documents:
+    type: inline_entity_form_complex
+    weight: 26
+    region: content
+    settings:
+      form_mode: default
+      override_labels: false
+      label_singular: ''
+      label_plural: ''
+      allow_new: true
+      allow_existing: false
+      match_operator: CONTAINS
+      allow_duplicate: false
+      collapsible: false
+      collapsed: false
+      revision: false
```

Comment:
I see in oe_content its revision true and removed_reference keep, but.. 
Why keep these entities if you can't select any existing ones ?

Reply:
@brummbar do you have any insight on this ?